FrostCord/FrostCord
# Fros stream implement

### Testing Instructions

1. npm i to make sure all dependancies are working properly. 
2. Open and navigate to a media channel 
3. Click join to start chat, ensure that when you get there your display name isnt appearing before you join
4. Turn on camera and mic and screen share to ensure they start. 
5. Open another browser and repeat steps 
6. Navigate to a chat room to make sure you can hear yourself
7. From the room its self turn off the camera and screen share controls, and hten turn them back on from the sidebar controls
8. end call from sidebar controls
9. switch on your mic from sidebar tool bar, join room again 
10. deafen the audio
11. end call from controls in room
